lib/honeybadger/request_id.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Honeybadger.RequestId do
+  @key __MODULE__
```

Comment:
This is private and shouldn't show up in documentation:

```suggestion
defmodule Honeybadger.RequestId do
  @moduledoc false
  
  @key __MODULE__
```